src/views/StatsView.vue

```diff
@@ -76,6 +76,12 @@
           </h2>
           <stats-leaderboard :loading="loadingData" :season="selectedSeason" />
         </template>
+
+        <!-- Usage stats -->
+        <div id="usage-stats-section">
+          <h2 class="text-h2 mt-8 mb-4">Site Usage</h2>
+          <stats-usage-chart :season="selectedSeason" />
```

Comment:
I like that because the `<stats-usage-chart>` is specialized for the specific line graphs we're showing, we can just send it a season object and the `StatsView` needs 0 additional logic, just to pass the already-selected season